slatedb/src/tablestore.rs

```diff
@@ -80,6 +82,7 @@ impl TableStore {
             PathResolver::new(root_path),
             Arc::new(FailPointRegistry::new()),
             block_cache,
+            false,
```

Comment:
noob question:  what is the difference between usage of `new` and `new_with_fp_registry` that makes one of them has a default as false ?